CHANGELOG.md

```diff
@@ -528,820 +542,6 @@ Notable fixes include:
 - Fix for custom logo in header
 - Fall back to VMRC desktop client if no NPAPI plugin is available
 - Default Filters can be saved or reset
-
-# Darga
```

Comment:
Darga should not be removed.  The Changelog is an accumulation of changes over time